index.html

```diff
@@ -46,38 +45,68 @@ <h1 class="display-4"></h1>
     </div>
     </div>
 
-<div class="d-flex justify-content-center">
-<h1>Main Entree's</h1>
+<div class="d-flex container">
+<div class="center-block">
+  <h2>Main Entree's</h2>
```

Comment:
**Main Entree's** looks like it is located within the jumbotron on my screen.
![Screenshot (5)](https://user-images.githubusercontent.com/25779736/98649084-01a2ba00-22ec-11eb-9903-12bcf14da168.png)

Reply:
Thank you!! And yes I am trying to see what can be done to move Main Entree's down and fix the reservation part to align with the other objects. Thank you again for your feedback Liz!